### PR TITLE
1017: Upgrade xcode 15

### PR DESCRIPTION
**Pull request by f1sh1918**
### Short description

Since we won't be able to publish ios apps after 29th of April, with XCode 14, we have to upgrade to xcode 15

### Proposed changes

- upgrade to xcode15

### Resolved issues

Fixes: #1017

---

### CODE REVIEW

File: .circleci/src/workflows/commit_main.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 when:
   and:
-    - << pipeline.parameters.run_commit >>
-    - equal: [main, << pipeline.git.branch >>]
+    - << pipeline.parameters.run_commit_main >>
+#    - equal: [main, << pipeline.git.branch >>]
```
Comment (steffenkleinle) [inline .circleci/src/workflows/commit_main.yml:4]: I guess this should be reverted?

File: .circleci/src/@common.yml [hunk 2]
```diff
@@ -5,6 +5,10 @@ parameters:
     default: true
     type: boolean
 
+  run_commit_main:
+    default: false
+    type: boolean
+
```
Comment (steffenkleinle) [inline .circleci/src/@common.yml:11]: Why is this needed?

File: .circleci/src/workflows/commit_main.yml (hunk 1 above)
Comment (f1sh1918) [reply .circleci/src/workflows/commit_main.yml]: i did it was just for testing

File: .circleci/src/@common.yml (hunk 2 above)
Comment (f1sh1918) [reply .circleci/src/@common.yml]: it was just for testing. sorry didn't draft the pr

**Comment by f1sh1918** [comment]
now the pr is ready @steffenkleinle 
https://app.circleci.com/pipelines/github/digitalfabrik/lunes-app/3503/workflows/299f3979-2b1d-44e5-9ac3-afc95ccfe4f7
Ios builds also work now

**Review by steffenkleinle — APPROVED** [review]
Looks good
